create-users-projects.yml

```diff
@@ -9,3 +9,6 @@
         - username: jkeating
           email: "{{ username + '@us.ibm.com' }}"
           isadmin: True
+        - username: mctaylor
+          email: "{{ mctaylor + '@us.ibm.com' }}"
```

Comment:
`mctaylor` is assumed to be a variable there, not a literal string. I think you want to either use `username` as the variable, or quote `'mctaylor'`.